tests/bluetooth/tester/src/audio/btp_bap_broadcast.c

```diff
@@ -125,15 +125,30 @@ static void stream_started(struct bt_bap_stream *stream)
 	btp_send_bis_syced_ev(&broadcaster->address, broadcaster->broadcast_id, b_stream->bis_id);
 }
 
+static void btp_send_bis_stopped_ev(const bt_addr_le_t *address, uint32_t broadcast_id,
+				  uint8_t bis_id, uint8_t reason)
+{
+	struct btp_bap_bis_stopped_ev ev;
+
+	bt_addr_le_copy(&ev.address, address);
+	sys_put_le24(broadcast_id, ev.broadcast_id);
+	ev.bis_id = bis_id;
+	ev.bis_stop_reason = reason;
+
+	tester_event(BTP_SERVICE_ID_BAP, BTP_BAP_EV_BIS_STOPPED, &ev, sizeof(ev));
+}
+
 static void stream_stopped(struct bt_bap_stream *stream, uint8_t reason)
 {
 	struct btp_bap_broadcast_stream *b_stream = stream_bap_to_broadcast(stream);
 
 	LOG_DBG("Stopped stream %p with reason 0x%02X", stream, reason);
 
 	btp_bap_audio_stream_stopped(&b_stream->audio_stream);
-
 	b_stream->bis_synced = false;
+	broadcaster = &remote_broadcast_sources[b_stream->source_id];
```

Comment:
I think it's missing the variable declaration for `broadcaster`: 
```c
struct btp_bap_broadcast_remote_source *broadcaster;
```

Reply:
Missed that one, thanks